site/content/docs/custom-theme.md

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
Structural comments on overall piece:

- I would have a section, probably at the start about tailwind and its role in Flowershow
  - explain that Flowershow uses tailwindcss. Point people to our reference docs on this (do we have them) and/or tailwindcss docs.
  - Explain that: thus customization of fonts and colors is going to be done via tailwindcss and especially its config file (against link docs)
  - Explain where to find this file (try and make this generic e.g. file is called `tailwind.config.js` and located in the directory where your flowershow app is installed, usually a `.flowershow` subfolder of wherever you ran `npx flowershow install`.
- then have section on customizing fonts
  -  explain what this means e.g. we are going to change heading font or body font across whole site
    - maybe note that if they just want to change one section that should be done using html/tailwindcss (TODO: a future tutorial perhaps ...)
  - then proceed as normal
- ditto for customizing colors